src/server/object_services/object_server.js

```diff
@@ -770,6 +770,20 @@ async function read_node_mapping(req) {
     };
 }
 
+
+function _convert_rpc_params_to_bucket_policy_req(rpc_params) {
```

Comment:
Could you add JSDoc to the new functions you added?

Reply:
done